### ISSUE TITLE
keyboard navigation items

### ISSUE THREAD

**Issue by cseppan** [USEPA-OAQPS/emf#42]
This issue is for noting any places where UI can't be controlled solely with the keyboard.

* Clicking on a column name in a table sorts the table by that column. Not sure if this is currently accessible via keyboard.

* When viewing the results of a QA step, the File > Close menu option is grayed out. The only way to close the window is to click the Close box.

* Missing button shortcuts in View QA Step window (View Results, Export)

* If the currently open window has a button shortcut which is the same as the main menu (F, M, W, T, or H), you can't access the menu.

**Comment by cseppan** [comment]
* The Sort-Filter-Select buttons can't be activated with the Enter key, just the spacebar. Hitting Enter activates the default button for the window, usually Close.

* After using the filter option in the Dataset Type Manager window, no UI element has focus, so keyboard navigation doesn't work anymore. Note that other places that use this same UI like the Control Strategy Manager and the Control Program Manager don't have this problem.

**Comment by cseppan** [comment]
Ctrl+F6 navigates between open windows but the window doesn't always get focus.

**Comment by cseppan** [comment]
For the Show Datasets of Type pull-down in the Dataset Manager window, typing a letter immediately selects the first dataset type starting with that letter. See if this can be changed to just go to that letter in the pull-down and not select the item.
